app/controllers/interviews_controller.rb

```diff
@@ -1,14 +1,26 @@
 class InterviewsController < ApplicationController
   before_action :set_interview, only: [:show, :edit, :update, :destroy]
 
-  def index
+  def home
     @interviews = Interview.all
+    render json: @interviews
+  end
+
+  def index
+    @interviews = Interview.all  
```

Comment:
Do we need this query in index? We are already fetching all the interviews in home.

Reply:
Agreed, we don't need it the fetching now. I'll remove it